test/parallel/test-process-resource-usage.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const usage = process.resourceUsage();
+
+assert(usage.userCpuTimeUsedSec >= 0);
```

Comment:
Anyway, I am okay with this. Could you look at the comment which I left in the doc please?